src/C-DEngine/C-DCommunication/cdeQueuedSender/cde_QueuedSenderREST.cs

```diff
@@ -295,6 +314,15 @@ void sinkUploadDataCompleted(TheRequestData eResult)
                 {
                     TheCDEKPIs.IncrementKPI(eKPINames.QSReceivedTSM);
                     TheCDEKPIs.IncrementKPI(eKPINames.QKBReceived, eResult.ResponseBuffer.Length);
+                    if (MyTargetNodeChannel != null)
```

Comment:
And one more time here

Reply:
Please see the new changes in `TheCDEKpis`.